modules/RTC/TraceablePeerConnection.js

```diff
@@ -2901,9 +2901,7 @@ TraceablePeerConnection.prototype._processLocalSSRCsMap = function(ssrcMap) {
 
             // eslint-disable-next-line no-negated-condition
             if (newSSRCNum !== oldSSRCNum) {
-                if (!oldSSRCNum) {
-                    logger.error(`Overwriting SSRC for ${track} ${trackMSID} in ${this} with: `, newSSRC);
-                }
+                oldSSRCNum && logger.error(`Overwriting SSRC for ${track} ${trackMSID} in ${this} with: `, newSSRC);
```

Comment:
Fixing a mistake in the previous commit, too lazy to open another PR for this :)